lib/Toolbar/Toolbar.js

```diff
@@ -5,7 +5,7 @@ import { Divider } from './Divider';
 function Toolbar({ children, className, ...rest }) {
   return (
     <div
-      className={`flex items-center border-0 border-b border-solid border-neutral-90 px-4 py-1 bg-white ${className}`}
+      className={`flex items-center border-0 border-b border-solid border-neutral-90 px-4 bg-white ${className}`}
```

Comment:
🚩 I think I originally added this padding as it was required when the text type dropdown button was on its focus state, worth checking whether this still looks ok 👍

Reply:
![Screenshot 2020-06-03 at 11 59 35](https://user-images.githubusercontent.com/12775966/83629295-be505c80-a591-11ea-9a2f-605b214c2690.png)
he's good!